lib/atomic/activities.ex

```diff
@@ -176,21 +177,45 @@ defmodule Atomic.Activities do
   end
 
   @doc """
-  Creates a activity.
+  Creates an activity and its respective post.
+  All in one transaction.
 
   ## Examples
 
+      iex> create_activity(%{field: value, ~N[2020-01-01 00:00:00]})
+      {:ok, %Activity{}}
+
       iex> create_activity(%{field: value})
       {:ok, %Activity{}}
 
+      iex> create_activity(%{field: bad_value, ~N[2020-01-01 00:00:00]})
+      {:error, %Ecto.Changeset{}}
+
       iex> create_activity(%{field: bad_value})
       {:error, %Ecto.Changeset{}}
 
   """
   def create_activity(attrs \\ %{}) do
```

Comment:
```suggestion
  def create_activity_with_post(attrs \\ %{}) do
```
And create a simpler function called `create_activity`.
(Same for announcements)